src/context/gallery/index.ts

```diff
@@ -37,6 +37,8 @@ const gallery: Service<Gallery> = context => {
           suggestedTags: [],
           createdAt: new Date(),
           updatedAt: new Date(),
+          date: new Date(),
+          name: originalname,
```

Comment:
I was thinking we could use fileInfo.originalname in place of this name prop. Seems more accurate. Let's remove name altogether.